exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -1564,3 +1576,40 @@ func (ku *Kucoin) GetFuturesContractDetails(ctx context.Context, item asset.Item
 	}
 	return resp, nil
 }
+
+// UpdateOrderExecutionLimits updates order execution limits
+func (ku *Kucoin) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item) error {
+	if !ku.SupportsAsset(a) {
+		return fmt.Errorf("%w %v", asset.ErrNotSupported, a)
+	}
+
+	if a != asset.Spot {
```

Comment:
I'm going to request that you add all supported assets, all API endpoints have been implemented